src/lib/replication/service/raft_repl_service.cpp

```diff
@@ -383,7 +384,22 @@ void RaftReplService::load_repl_dev(sisl::byte_view const& buf, void* meta_cooki
     }
 
     if (rd_sb->destroy_pending == 0x1) {
-        LOGINFOMOD(replication, "ReplDev group_id={} was destroyed, skipping the load", group_id);
+        LOGINFOMOD(replication, "ReplDev group_id={} was destroyed, reclaim the stale resource", group_id);
+        // if we do not add the repl_dev to m_rd_map, it will not be permanently destroyed since gc thread finds the
+        // pending destroy repl_dev only from m_rd_map. so, we should try to reclaim all the repl_dev stale resources
+        // here.
+
+        // 1 since we permanantly destroy the repl_dev here, it will not join_raft group where raft_server will be
+        // created. hence , no need to detroy it through nuraft_mesg, where raft_server will be shutdown.
+        // 2  m_raft_config_sb will be destroyed in raft_group_config_found() method if repl_dev is is not found, so
+        // skip it.
+
+        // 3 logdev will be destroyed in delete_unopened_logdevs() if we don't open it(create repl_dev) here, so skip
+        // it.
```

Comment:
we need do nothing here,  since if we do not create the repl_dev , [the related log_dev will not be opened](https://github.com/eBay/HomeStore/blob/4933dfb18abf6e0490e87ff280d8e80b72856d1d/src/lib/replication/log_store/home_raft_log_store.cpp#L100), as a result , the log dev will be eventually detroyed at 

https://github.com/eBay/HomeStore/blob/4933dfb18abf6e0490e87ff280d8e80b72856d1d/src/lib/replication/service/raft_repl_service.cpp#L176